Makefile

```diff
@@ -91,4 +91,4 @@ typecheck-python:
 changeset:
 	@yarn changeset
 
-ci: lint typecheck test
+ci: lint typecheck test check-dependencies
```

Comment:
Nit: this is closer to a "lint" task and will generally succeed much faster than `make test`, so I'd be inclined to have it run before `test`.

Reply:
I did it off the order I saw the tests in CI, but those are probably randomized. I'm OK moving it up.